src/main/java/com/sap/oss/phosphor/fosstars/data/github/UsesOwaspDependencyCheck.java

```diff
@@ -139,6 +140,7 @@ private static ValueSet checkMaven(LocalRepository repository) throws IOExceptio
         OwaspDependencyCheckUsageValue usage = OWASP_DEPENDENCY_CHECK_USAGE.value(visitor.usage());
 
         OwaspDependencyCheckCvssThresholdValue threshold = visitor.threshold()
+            .filter(value -> CVSS.MIN <= value && value <= CVSS.MAX)
```

Comment:
Could you please confirm?
- If lets say that `filter()` returns empty list.
- Then would `orElse` return the `notSpecifiedValue`?
- What is the value = 12?

Reply:
I guess the corresponding test is actually testing that?

Reply:
`filter()` doesn't return a list, it returns an `Optional`. The predicate in `filter()` checks that `value` is a valid CVSS, otherwise it returns an empty `Optional` that results in a not specified threshold. This check is necessary because `failBuildOnCVSS` tag can contain any number.

> I guess the corresponding test is actually testing that?

Yes, it is covered in the new test case.